Jenkinsfile_CNP

```diff
@@ -64,5 +64,6 @@ withPipeline("nodejs", product, component) {
 
   after('functionalTest:preview') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test/functional/output/*'
+    env.CCD_ADMIN_URL="https://ccd-admin-web.aat.platform.hmcts.net"
```

Comment:
you will need to separate out `preview` and `aat` steps, this is the correct URL for AAT, not preview

Reply:
yes, you're right